src/plugins/webSocketClient.ts

```diff
@@ -68,8 +70,13 @@ export class WebSocketClient {
             return
         }
 
+        // resolve promise if it exists
+        if (wait?.resolve && 'result' in data) {
```

Comment:
we check if the wait exists a couple of lines before. So there no need for the null check operator.